### ISSUE TITLE
investigate possibility to use go ios to check device availability

### ISSUE THREAD

**Issue by vdelendik** [zebrunner/mcloud-agent#266]
so far we try to start containers for every registered ios device. for those which are not connected we start and exit.
let's review the steps and maybe incorporate verification before we start 3 containers using go ios.
btw, go ios should be deployed additionally onto the linux/mac server.

**Comment by vdelendik** [comment]
closing as won't fix. udev rule manages connect/disconnect.
it is impossible to use go ios on Linux as usbmuxd service is disabled/masked
